README.md

```diff
@@ -156,7 +190,18 @@ For JDK 21 (recommended mockito version 5.7.0, we need byte-buddy that is equal
       </dependency>
   ```
 
-- Gradle
+- Gradle for JDK 23
+
+ ``` 
+       testImplementation group: 'org.mockito', name: 'mockito-core', version: '5.7.0'
+       testImplementation group: 'org.mockito', name: 'mockito-inline', version: '5.2.0'{
+		exclude group: 'net.bytebuddy', module: 'byte-buddy
+	}
+       // byte-buddy is an optional dependency. Mockito brings it, but an old version can cause problems. We need at least 1.12.9
```

Comment:
@bhuvan-sapientai , change the version here in the comment too please

Reply:
Changed. Thanks Andre